### PR TITLE
Debug/post

### PR DESCRIPTION
**Pull request by shanekwarning**
### Description
This request will update our submission form by adding a confirmation window. This fixes timing issues. It will also add the refetch method to our useQuery allowing us to refetch data after a medicine has been added. 

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)

### Checklist:

- [ x] I have performed a self-review of my own code
- [ x] I have made corresponding changes to the documentation
- [ x] My changes generate no new warnings
- [ x] I have added tests that prove my fix is effective or that my feature works
- [ x] New and existing unit tests pass locally with my changes
- [ x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: src/Components/App/App.tsx [hunk 1]
```diff
@@ -14,7 +14,8 @@ import { useQuery } from "@apollo/client";
 const App = () => {
   const [user, setUser] = useState<User>()
   const [meds, setMeds] = useState<QueryRx[]>([])
-  const { loading, error, data } = useQuery(GET_USER)
+  const { loading, error, data, refetch } = useQuery(GET_USER)
```
Comment (emmacbd) [inline src/Components/App/App.tsx:17]: Nice call with the refetch!

File: src/Components/Form/SearchForm.tsx [hunk 2]
```diff
@@ -4,9 +4,10 @@ import SubmissionForm from './SubmissionForm'
 
 type UserProps = {
   userID: number
+  refetch: any
```
Comment (angelewilliams) [inline src/Components/Form/SearchForm.tsx:7]: Not sure if we should keep the `any` type here

File: src/Components/Form/SubmissionForm.css [hunk 3]
```diff
@@ -16,4 +16,35 @@
     align-content: flex-start;
     align-items: center;
     justify-content: space-between;
+}
+
+.modal{
```
Comment (emmacbd) [inline src/Components/Form/SubmissionForm.css:21]: Thanks for adding this modal.

**Review by angelewilliams — APPROVED** [review]

**Review by emmacbd — APPROVED** [review]
Good work squashing this bug!

**Pull request merged**